packages/core-components-next/src/components/FileInput.vue

```diff
@@ -68,15 +73,16 @@
         variant="tonal"
         @click="expanded = false"
       />
-      <img :src="image" alt="Full image" />
+      <img :src="publicUrl || value.publicUrl" alt="Full image" />
     </VOverlay>
   </div>
 </template>
 
 <script lang="ts" setup>
-import { computed, ref } from 'vue';
+import { computed, onMounted, ref } from 'vue';
 import { type VFileInput } from 'vuetify/components';
 
+import { useStorageService } from '../composables/useStorageService';
```

Comment:
Since this is a separate lib, can we use injected $storageService ?

Reply:
Yes, it's already used within the useLoader